src/Components/Admin/EditUserForm.js

```diff
@@ -102,15 +60,6 @@ function EditUserForm({ dispatch, formState }) {
           onChange={dispatchRoleChange}
         />
       </QuestionWrapper>
-      <QuestionWrapper>
-        <StyledLabel htmlFor="role-select">
-          Which program(s) are they a part of?
-        </StyledLabel>
-        <ProgramSelect
-          onChange={dispatchProgramsChange}
-          value={formState.programs}
-        />
-      </QuestionWrapper>
```

Comment:
This could be useful for organization access management (a nice-to-have), but not needed for now. I can make a note to refer back to this code in the ticket, in case it ends up being done.